### ISSUE TITLE
vader redirects all print output to console in jupyter notebook

### ISSUE THREAD

**Issue by shermanash** [cjhutto/vaderSentiment#7]
just spent a few hours debugging this, finally narrowed it down to the cause--after importing vader using

"from vaderSentiment.vaderSentiment import sentiment as vaderSentiment" ,
any print function does not print to the ipython notebook - it redirects the print output to the terminal where the ipython notebook was launched from.

using ipython notebook 4.1.1 and python 2.7.11

**Comment by azikoss** [comment]
The same problem here on Python 2.7.11, ipython 4.1.0.

**Comment by azikoss** [comment]
This is fix the problem. Load the library in the way that it redirects the notebook's output back to the normal.

```
import sys
nb_stdout = sys.stdout
from vaderSentiment.vaderSentiment import sentiment as vaderSentiment
sys.stdout = nb_stdout
```

**Comment by mrbeann** [comment]
This is caused by `reload(sys)`, just delete it can help you fix this problem.

**Comment by cjhutto** [comment]
Thanks! 
The new update (with new pip install) fixes this issue.
We've also incorporated better compatibility support for Python 3 to address many of the encoding/decoding issues. The lexicon dictionary file has been encoded with UTF-8 unicode by default (I hope) for better cross-OS performance. I've also implemented code to automatically detect where the dictionary file is installed (as long as you didn't change it's location relative to where the actual "vaderSentiment.py" file got installed), so you no longer any need to manually put a copy of the dictionary file next to your python file (in the same folder)... from what I can tell, my implementation should work across OSs.
